cooking-app/src/components/Content/UserPanel/DisplayUserData.js

```diff
@@ -0,0 +1,247 @@
+import React from "react";
+import styled from "styled-components";
+import { useState, useEffect, useContext } from "react";
+import { UserDataContext } from "../../../App";
+import { IconButton, Button, Paper, Avatar } from "@mui/material";
+import PhotoCamera from "@mui/icons-material/PhotoCamera";
+import CloseIcon from "@mui/icons-material/Close";
+import { v4 } from "uuid";
+import {
+  storageErrorsCodes,
+  urlStorageAvatars,
+  urlStorageAvatarsCD,
+} from "../../../api/firebaseIndex";
+import { deleteObject, ref, uploadBytes } from "firebase/storage";
+import { storage, db } from "../../../api/firebase";
+import { doc, getDoc, updateDoc } from "firebase/firestore";
+//import { CustomAvatar } from "../../Header/CustomAvatar";
+
+// const defaultAvatar =
+// "https://firebasestorage.googleapis.com/v0/b/devs-project-edf3a.appspot.com/o/avatar%2Favatar%20default.jpg8d69a1ee-c52d-4004-83a4-d5efa733c5ab?alt=media&token=78be46ed-8666-41d2-b987-b8782d27da63";
+
+// const urlStorageAvatars =
+//   "https://firebasestorage.googleapis.com/v0/b/devs-project-edf3a.appspot.com/o/avatar%2F";
+// const urlStorageAvatarsCD =
+//   "?alt=media&token=447796ed-bf03-404d-902f-ab81082e8c0d";
+
+export const DisplayUserData = () => {
+  const CurrentUser = useContext(UserDataContext);
+  const [currentUserData, setCurrentUserData] = useState(null);
+  const [imageUpload, setImageUpload] = useState(null);
+  const [avatarUrl, setAvatarUrl] = useState(null);
+  const [avatarUrlId, setAvatarUrlId] = useState("");
+  const [docRefUser, setDocRefUser] = useState(null);
+
+  useEffect(() => {
+    //console.log("--useEffect CurrentUser-", CurrentUser);
+    setCurrentUserData(CurrentUser);
+  }, [CurrentUser]);
+
```

Comment:
wiem, ze rozmawialismy o tym, i nie ma dostepu do Contextu usera bez tego, tak? Nie pamietam gdzie byl problem. Ale gdyby uzyc walidacji czy jest user to cos tam..., a jak nie to loading lub null cyz jakos tak? tez mialam problem z dostepem do danych usera i jakos tak to rozwiazalam

Reply:
i jest jakis konflikt brancha develop w App

Reply:
i jest jakis konflikt brancha develop w App

Reply:
W App dodałam pole do userContext, potrzbuję avatarUrlID żeby usuwać niepotrzebne zdjęcia ze Storage